paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/PaymentOptionsStateMapper.kt

```diff
@@ -1,49 +1,48 @@
 package com.stripe.android.paymentsheet.viewmodels
 
-import androidx.lifecycle.LiveData
-import androidx.lifecycle.MediatorLiveData
-import androidx.lifecycle.distinctUntilChanged
 import com.stripe.android.model.PaymentMethod
 import com.stripe.android.paymentsheet.PaymentOptionsState
 import com.stripe.android.paymentsheet.PaymentOptionsStateFactory
 import com.stripe.android.paymentsheet.model.PaymentSelection
 import com.stripe.android.paymentsheet.model.SavedSelection
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.distinctUntilChanged
 
 internal class PaymentOptionsStateMapper(
-    private val paymentMethods: LiveData<List<PaymentMethod>>,
-    private val isGooglePayReady: LiveData<Boolean>,
-    private val isLinkEnabled: LiveData<Boolean>,
-    private val initialSelection: LiveData<SavedSelection>,
-    private val currentSelection: LiveData<PaymentSelection?>,
+    private val paymentMethods: StateFlow<List<PaymentMethod>>,
```

Comment:
If no default value is needed initially, you might want to use `SharedFlow` which doesn't require a initial value.

(not required in this PR but just FYI) Also If this flow is for triggering UI, you could wrap it around [Resource](https://github.com/stripe/stripe-android/blob/7e2e84fa62e0cf1828709cb5405f42f8d2680baf/identity/src/main/java/com/stripe/android/identity/networking/Resource.kt#L12), which has a [Status](https://github.com/stripe/stripe-android/blob/289fbc5df8e0428e2f55360b9ce3a681dc9961c8/identity/src/main/java/com/stripe/android/identity/networking/Status.kt#L10) to indicate if the resource if available or not(instead of setting its value to null or false), it's useful if this `stateFlow` is indicating a resource returned from a network call. I found it useful for trigging UI events, [here](https://github.com/stripe/stripe-android/blob/03ae82743f9f062cd0a096954c575e2c71402d37/identity/src/main/java/com/stripe/android/identity/ui/LoadingScreen.kt#L45) is an example consuming it